mp/src/game/client/momentum/ui/HUD/hud_keypress.cpp

```diff
@@ -221,6 +228,21 @@ void CHudKeyPressDisplay::Paint()
         surface()->DrawSetTextPos(GetTextCenter(m_hWordTextFont, m_pwDuck), duck_row_ypos);
         surface()->DrawPrintText(m_pwDuck, wcslen(m_pwDuck));
     }
+    // Add M1 and M2 buttons
+    if (nButtons & IN_ATTACK)
+    {
+        CHECK_INPUT_P(IN_ATTACK);
+        int text_m1 = GetTextCenter(m_hTextFont, m_pwM1) - (UTIL_ComputeStringWidth(m_hTextFont, m_pwM1)) * 3.5;
```

Comment:
The variable name here is not descriptive, use something like `xposM1` instead.

Also I'm not a fan of the random `* 3.5` scalar happening here. Is there any way this can be more absolute? It's only two characters, this seems like it's overly complicated for positioning them.

Reply:
The issue with the positioning was that I didn't really have a sense of scale for how much changing the value would change the x position of the element so I just experimented with it until I reached something that looked good enough. I will try to figure out a way to define it better.

Reply:
I found out that using `WordTextFont` instead of `TextFont` makes the positioning more consistent without the need for arbitrary factors (using one single factor across all elements is sufficient).